static/js/modules/filter-panel.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var FilterSet = require('./filter-set').FilterSet;
+var accessibility = require('fec-style/js/accessibility');
+
+var defaultOptions = {
+  body: '.filters',
+  form: '#category-filters',
+  toggle: '#filter-toggle'
+};
+
+function FilterPanel(options) {
+  this.isOpen = false;
+  this.options = _.extend({}, defaultOptions, options);
+
+  this.$body = $(this.options.body);
+  this.$form = $(this.options.form);
+  this.$toggle = $(this.options.toggle);
+
+  this.$toggle.on('click', this.toggle.bind(this));
+
+  this.filterSet = new FilterSet(this.$form).activate();
+  if (!_.isEmpty(this.filterSet.serialize())) {
+    this.show();
+  }
+
+  this.adjust();
+}
+
+FilterPanel.prototype.adjust = function() {
+  if ($('body').width() > 768) {
+    this.show();
+  } else if (!this.isOpen) {
+    this.hide();
+  }
+};
+
+FilterPanel.prototype.show = function() {
+  this.$body.addClass('is-open');
+  this.$toggle.addClass('is-active');
+  this.$toggle.find('.filters__toggle__text').html('Hide filters');
```

Comment:
If the style of the filter panel changes and it uses a different class from `filters__toggle__text`, would this break?

I like separating my classes that are for JS hooks from my classes that are for styles, but will also respect what has been used and works in this codebase.